net/wfb-ng/files/wfb-ng.init

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=10
+
+USE_PROCD=1
+WFB_INIT="/usr/sbin/wfb-ng.sh"
+
+start_service() {
+        if [ -x $WFB_INIT ]; then
```

Comment:
this makes me wonder what does this file do? calling wfb binraries with some custom arguments?

Reply:
If filled by auto generated init script when used in cluster mode. Central server will generate init scripts for each node in cluster according to cluster topology. For example:

``` .sh
#!/bin/bash
set -emb

export LC_ALL=C

_cleanup()
{
  plist=$(jobs -p)
  if [ -n "$plist" ]
  then
      kill -TERM $plist || true
  fi
  exit 1
}

trap _cleanup EXIT

iw reg set US

# init wlx9803cfd153c6
if which nmcli > /dev/null && ! nmcli device show wlx9803cfd153c6 | grep -q '(unmanaged)'
then
  nmcli device set wlx9803cfd153c6 managed no
  sleep 1
fi

ip link set wlx9803cfd153c6 down
iw dev wlx9803cfd153c6 set monitor otherbss
ip link set wlx9803cfd153c6 up
iw dev wlx9803cfd153c6 set channel 161 HT20
iw dev wlx9803cfd153c6 set txpower fixed 100

# init wlx9803cfd153d5
if which nmcli > /dev/null && ! nmcli device show wlx9803cfd153d5 | grep -q '(unmanaged)'
then
  nmcli device set wlx9803cfd153d5 managed no
  sleep 1
fi

ip link set wlx9803cfd153d5 down
iw dev wlx9803cfd153d5 set monitor otherbss
ip link set wlx9803cfd153d5 up
iw dev wlx9803cfd153d5 set channel 161 HT20
iw dev wlx9803cfd153d5 set txpower fixed 100

# gs_video
wfb_rx -f -c 192.168.1.2 -u 10000 -p 0 -i 7669206 -R 2097152 wlx9803cfd153c6 wlx9803cfd153d5 &

# gs_mavlink
wfb_rx -f -c 192.168.1.2 -u 10001 -p 16 -i 7669206 -R 2097152 wlx9803cfd153c6 wlx9803cfd153d5 &
wfb_tx -I 11002 -R 2097152 -Q -P 10 wlx9803cfd153c6 wlx9803cfd153d5 &

# gs_tunnel
wfb_rx -f -c 192.168.1.2 -u 10002 -p 32 -i 7669206 -R 2097152 wlx9803cfd153c6 wlx9803cfd153d5 &
wfb_tx -I 11004 -R 2097152 -Q -P 20 wlx9803cfd153c6 wlx9803cfd153d5 &


echo "WFB-ng init done"
wait -n
```